purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1436,7 +1436,7 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
 
     private fun getPurchasesUpdatedListener(): BillingAbstract.PurchasesUpdatedListener {
         return object : BillingAbstract.PurchasesUpdatedListener {
-            override fun onPurchasesUpdated(purchases: List<@JvmSuppressWildcards PurchaseWrapper>) {
+            override fun onPurchasesUpdated(purchases: List<PurchaseDetails>) {
```

Comment:
BillingAbstract.PurchasesUpdatedListener is in Kotlin now, so we don't need these JvmSuppressWildcards